composer.json

```diff
@@ -44,13 +45,12 @@
     "require-dev": {
         "fakerphp/faker": "^1.9.1",
         "kitloong/laravel-migrations-generator": "^6.8",
-        "mll-lab/laravel-graphiql": "^3.1",
         "mockery/mockery": "^1.4.4",
         "nunomaduro/collision": "^7.0",
-        "phpstan/phpstan": "^1.4.7",
-        "phpunit/phpunit": "10.2.1",
+        "phpunit/phpunit": "10.0.1",
```

Comment:
@FredPeal why are you downgrading this?

Reply:
Because I'm usign power shell , the current version has a bug with power shell.